### PR TITLE
[Merged by Bors] - chore: remove more variable at the end of the imports

### PR DESCRIPTION
**Pull request by adomani**
---

[Open in Gitpod](https://gitpod.io/from-referrer/)

### CODE REVIEW

File: Mathlib/Topology/Instances/RatLemmas.lean [hunk 1]
```diff
@@ -72,7 +72,7 @@ theorem not_secondCountableTopology_opc : ¬SecondCountableTopology ℚ∞ := by
   exact not_firstCountableTopology_opc inferInstance
 
 instance : TotallyDisconnectedSpace ℚ := by
-  clear p q s t
+  clear p s
```
Comment (Ruben-VandeVelde) [inline Mathlib/Topology/Instances/RatLemmas.lean:75]: Is that still needed?

**Review by Ruben-VandeVelde — APPROVED** [review]

File: Mathlib/Topology/Instances/RatLemmas.lean (hunk 1 above)
Comment (adomani) [reply Mathlib/Topology/Instances/RatLemmas.lean:75]: I have not tried deleting that, but the variables are in scope: the new behaviour does not apply to def's and def-adjacent declarations.

**Comment by jcommelin** [comment]
Thanks :tada:

bors merge

File: Mathlib/Topology/Instances/RatLemmas.lean (hunk 1 above)
Comment (adomani) [reply Mathlib/Topology/Instances/RatLemmas.lean:75]: I went back and looked at removing `clear` and it is not possible without more changes.  I did not investigate all the way, but it seems like the `wlog` step a little further down generates an implication that uses `p` and `s`, if they are in the local context.

**Pull request closed without merging**